docs/documentation-api/3 - Document Generation Module/3 - mobee-document-configuration.md

```diff
@@ -6,7 +6,52 @@ To utilize the module, users must meet the following prerequisites:
 
 - Possession of a Mobee license.
 - Assignment of either the "Mobee User" or "Mobee Administrator" permission set.
-- Activation of the "Flow User" option if needed.
+- Enable User External Credentials Through User Profiles
+
+## Access Authorization for Document Generation
+
```

Comment:
Write a small paragraph about why we need this authorization that indicates that Mobee uses an external service that generates the documents and that this service is secured with a Token authentication method.